captum/optim/_core/loss.py

```diff
@@ -75,21 +239,30 @@ def __call__(self, targets_to_values: ModuleOutputMapping) -> torch.Tensor:
             activations.size(2), activations.size(3), self.x, self.y
         )
 
-        return activations[:, self.channel_index, _x : _x + 1, _y : _y + 1]
+        if self.batch_index is None:
+            return activations[:, self.channel_index, _x : _x + 1, _y : _y + 1]
+        else:

```

Comment:
There may be a more graceful way to do this:
```
x = torch.ones(1,1,4,4)
x[batch_index : batch_index + 1].shape  # torch.Size([1, 1, 4, 4])
x[None : None].shape  # torch.Size([1, 1, 4, 4])
```

Reply:
Thanks! Using this instead